bots/images/scripts/rhel-x.install

```diff
@@ -4,3 +4,7 @@ set -e
 
 /var/lib/testvm/fedora.install --rhel --skip cockpit-kubernetes \
                                --skip cockpit-docker "$@"
+
+# HACK: https://bugzilla.redhat.com/show_bug.cgi?id=1600452
+# packagekitd spins at 100% CPU on startup
+systemctl mask packagekit.service
```

Comment:
This kind of problem seems likely to be fixed relatively soon...  Do we have a way to track upstream bugs so that we don't lose track of hacks like this?

Reply:
With cockpit being "everyone's QA", I file a lot of them :-) For others I subscribe to these, so that I know when they get fixed, and can also help to verify them. Most of these hacks get tracked by naughty overrides that get cleaned up automatically, but this doesn't work here.